airgun/views/provisioning_template.py

```diff
@@ -81,7 +80,7 @@ class association(SatTab):
         applicable_os = MultiSelect(id='ms-provisioning_template_operatingsystem_ids')
 
         @View.nested
-        class hg_environment_combination(RemovableWidgetsItemsListView):
+        class vaild_hostgroups(RemovableWidgetsItemsListView):
```

Comment:
@shubhamsg199 Could you fix this typo for 6.11.z, so you don't cherrypick of your typo fix PR
```suggestion
        class valid_hostgroups(RemovableWidgetsItemsListView):
```